doc/cli/vast.md

```diff
@@ -32,13 +32,20 @@ for `vast start`.
 
 ## Configuration
 
-In addition to command options, a configuration file `vast.conf` allows for
-persisting option values and tweaking system parameters. Command line options
-always override configuration file values.
+In addition to command options, a YAML configuration file `vast.yaml` allows
+for persisting option values and tweaking system parameters. Command line
+options always override configuration file values.
 
-During startup, `vast` looks for a `vast.conf` in the current directory. If
-the file does not exist, `vast` then attempts to open `PREFIX/etc/vast.conf`
-where `PREFIX` is the installation prefix (which defaults to `/usr/local`).
+During startup, `vast` looks for a `vast.yaml` or `vast.yml` in the following
```

Comment:
Let's just not mention `yml` at all, but rather silently support it as a nicety. This allows you to specify full paths below, lowering the complexity for the reader.